package.json

```diff
@@ -80,6 +80,7 @@
     "core-js": "3.6.5",
     "cross-env": "7.0.3",
     "deep-map-keys": "^2.0.1",
+    "dot-env": "0.0.1",
```

Comment:
do we need this?

Reply:
No, we don't. I'll remove it. I was fixing the process error, might've left it in there after debugging.